python/ray/serve/backend_worker.py

```diff
@@ -369,6 +380,11 @@ def reconfigure(self, user_config) -> None:
                                          BACKEND_RECONFIGURE_METHOD)
             reconfigure_method(user_config)
 
+    async def _update_backend_configs(self, backend_configs):
+        for backend_tag, config in backend_configs.items():
+            if backend_tag == self.backend_tag:
```

Comment:
leave a note to remove this once we can poll for a specific key?

Reply:
SGTM!